pkgs/development/python-modules/mailrise/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  pkgs,
+  lib,
+  buildPythonPackage,
+}:
+buildPythonPackage {
+  name = "mailrise";
+  pname = "mailrise";
+  version = "1.4.0";
```

Comment:
Either change this to `1.4.0-unstable-2024-01-14` or use the git tag `1.4.0` instead of the commit hash of the most recent commit.